update.xml

```diff
@@ -753,7 +766,7 @@
 
         <z2_plus>
             <maintainer>Yashodhan Sawardekar</maintainer>
-            <devicename>ZUK Z2 Plus</devicename>
+￼            <devicename>ZUK Z2 Plus</devicename>
```

Comment:
there's still an issue on this line 
not sure that char is this "￼ " just remove it

Reply:
where i don't get u @gwolf2u

Reply:
done @gwolf2u

Reply:
sry for inconvenience next time i will use my pc :)